department-of-veterans-affairs/vets-website
# Add "over 65" page to pensions

## Summary

On Step 3 Health and employment information,
ensure question matches the designs for:
Are you 65 years old or older?
as shown on the designs.

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the pension_form_enabled flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run vets-website and vets-api

## How to verify
1. Go to http://localhost:3001/pension/application/527EZ in your browser
2. Select 'Start your application without signing in'
3. Proceed to Chapter 3 by filling out the required fields on Chapter 1 and Chapter 2. Note the date of birth you enter.
4. Verify there's a boolean question 'Are you 65 years old or older?' matching the acceptance criteria
     - The field should default to true if the provided date of birth was over 65 years ago, and otherwise default to false
     - The field should be able to be edited
5. Verify that the employment questions at the end of section 3 are skipped if the veteran's date of birth is 65 years ago and  'Are you 65 years old or older?' is true (if either is false, the questions will still be asked) 

## Related issue(s)

- [Link to GitHub Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69757)
- [Link to Epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done



## Screenshots

| After |
| ------- | 
| <img width="691" alt="Screenshot 2023-12-04 at 3 31 25 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/b041a83b-4322-4ace-a3a1-b1b83a241fa8"> |

## What areas of the site does it impact?

Pension Benefits

## Acceptance criteria

- [Link to GitHub Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69757)

### Quality Assurance & Testing


### Error Handling


### Authentication
